NetworkConfigurations/NetworkDiscoveryAndManagementSubCategories/STP_SpanningTreeProtocol.md

```diff
@@ -0,0 +1,277 @@
+---
+title: STP (Spanning Tree Protocol)
+grand_parent: Network Configuration
+parent: Network Discovery and Management
+nav_order: 1
+layout: default
+---
+
+# STP (Spanning Tree Protocol)
+
+## Introduction
+
+In this guide, we will quickly explain what the
```

Comment:
why is this sentence broken into three lines. What does the final doc look like?

Reply:
The image below is what the final doc would look like. 

![Screenshot 2024-04-03 114236](https://github.com/dentproject/docs.dent.dev/assets/140857681/763ac94b-368c-461c-abfa-2c630ac8f11c)

I separated the lines because I thought it would help with readability. The image below is what it would look like combined. Let me know what you would prefer.

![Screenshot 2024-04-03 115116](https://github.com/dentproject/docs.dent.dev/assets/140857681/5f7d9fe0-6eae-4239-a4f5-6dd16b65e93c)